testbeam_analysis/tools/plot_utils.py

```diff
@@ -1055,3 +1055,37 @@ def efficiency_plots(hit_hist, track_density, track_density_with_DUT_hit, effici
         output_fig.savefig()
     else:
         logging.warning('Cannot create efficiency plots, since all pixels are masked')
+
+
+def plot_track_angle(input_track_slopes, output_data, output_pdf):
+    direction = ['x', 'y', 'z']
+
+    with PdfPages(output_pdf) as output_fig:
+        i = 0  # counting number of progressed DUTs
+        for key in input_track_slopes:
+            for j in (0, 1):  # iterate over slopes
+                min = np.min(input_track_slopes[key][j])
+                max = np.max(input_track_slopes[key][j])
+                edges = np.arange(min - 0.00005, max + 0.00005, 0.0001)
+                bin_center = (edges[1:] + edges[:-1]) / 2.0
+
+                sigma = np.std(np.arctan(input_track_slopes[key][j]))  # initial guess for sigma
+                mean = np.mean(np.arctan(input_track_slopes[key][j]))  # initial guess for mean
+
+                histo = plt.hist(np.arctan(input_track_slopes[key][j]), edges, label='Angular Distribution', color='#4361ba')
```

Comment:
To unify the colors one could use a 'c' color here. Check [matplotlib 2 color nfo](http://matplotlib.org/api/colors_api.html).